README.md

```diff
@@ -17,23 +17,22 @@ function MyButton () {
 }
 MyButton.prototype = Object.create(Nanocomponent.prototype)
 
-MyButton.prototype._render = function (color) {
-  this._color = color
+MyButton.prototype._render = function () {
```

Comment:
just because you can access `this.props` though doesn't mean there's a single prototype involved. yeah, `this` _can_ be weird, what do you think about passing `(ctx)` which has `ctx.{state,props,oldState}`. Seems a bit weird to me though, when we can just use `this`